tests/unit/insertTestResult.unitTest.ts

```diff
@@ -3211,11 +3211,17 @@ describe('insertTestResult', () => {
     "when inserting a 'fail' Specialist test with blank certificate number",
     () => {
       it('should not throw an error', () => {
-        const testResult = testResultsPostMock[13];
-        expect.assertions(1);
-        expect(ValidationUtil.validateInsertTestResultPayload(testResult)).toBe(
-          true,
-        );
+        try {
```

Comment:
As mentioned in the jira ticket, I think there should be at least 1 example for each AC being met in the unit tests.